nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -6,6 +6,8 @@
 
 - The default PHP version has been updated to 8.3.
 
+- `bcachefs` is now enabled in the installation media with a suitable kernel.
```

Comment:
```suggestion
- `bcachefs` is now enabled in the installation media with the latest kernel available.
```

Reply:
I think we can see 25.05 releases with 6.12 as the default, and hopefully bcachefs module can drop its version override. In that case we just go back to the version check (>=6.7) so `a suitable kernel` will be more appropriate.

But then it turned out currently it is quite difficult to check if the `config.boot.kernelPackages` is actually `pkgs.linuxPackages_latest`, so this PR is no longer the one-liner I was expecting. Might be better to actually wait for those blockages to be fixed first before continuing.